test/suite/test_cursor15.py

```diff
@@ -55,10 +55,10 @@ def test_cursor15(self):
         self.reopen_conn()
 
         # We don't restart the database between runs to exercise that
-        # `read_once` plays nice with cursor caching. Because the eviction
-        # statistics this test asserts are counters, we run the `read_once`
-        # table scan first and assert the counters stay zero. The second run
-        # without `read_once` should increment the eviction statistics.
+        # `read_once` plays nice with cursor caching. Note, there are no
+        # reliable statistics to check that `read_once` behaved as expected. The
+        # future may introduce a statistic for `WT_READ_WONT_NEED` being
+        # exercised. That may work as a suitable side-effect to observe here.
```

Comment:
minor nits: 'behaved' -> 'behaves'. 
Remove the ``.

Reply:
Done.